internal/prompts/app_select.go

```diff
@@ -1497,7 +1090,6 @@ func flatTeamSelectPrompt(
 // TeamAppSelectPrompt prompts the user to select an app from a specified team environment,
 // returning the selected app. This app might require installation before use if `status == ShowAllApps`.
 func TeamAppSelectPrompt(ctx context.Context, clients *shared.ClientFactory, env AppEnvironmentType, status AppInstallStatus) (SelectedApp, error) {
```

Comment:
👁️‍🗨️ With a flattened app selector we are so close to removing `TeamAppSelectPrompt` altogether to prefer passing the `AppEnvironmentType` in calling code as `ShowAllEnvironments` I hope.

Reply:
🧪 I am planning to update tests with this in mind, preferring to focus on logic in the `flatAppSelectPrompt` using various "environments".

Reply:
🗣️ The removal of `TeamAppSelect` might be in https://github.com/slackapi/slack-cli/pull/159!